engines/scumm/he/mixer_he.h

```diff
@@ -0,0 +1,211 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#if !defined(SCUMM_HE_MIXER_HE_H)
+#define SCUMM_HE_MIXER_HE_H
+
+#include "scumm/he/sound_he.h"
+#include "scumm/he/intern_he.h"
+#include "scumm/resource.h"
+
+#include "common/util.h"
+#include "common/file.h"
+#include "common/debug.h"
+#include "common/memstream.h"
+
+#include "audio/decoders/adpcm.h"
+#include "audio/audiostream.h"
+#include "audio/decoders/raw.h"
+#include "audio/decoders/wave.h"
+#include "audio/decoders/mp3.h"
+#include "audio/decoders/vorbis.h"
+#include "audio/decoders/flac.h"
+
+namespace Scumm {
+
+#define CHANNEL_EMPTY_FLAGS        0x00000000
```

Comment:
One of the more readable ways is to write `(1 << 1), (1 << 2)` etc. But it is matter of taste